### DISCUSSION TITLE
Edit code on mobile

### DISCUSSION THREAD

**Discussion by fsubp** [community/community#16218] [Mobile] [answered]
I want to suggest the option to edit code on the mobile app. Would be great to see and good for people who aren't always at home but still need to edit small stuff quick.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.63.0 (66585841)
Device: iPhone 11 (iOS 15.5)
</details>

**Comment by amrehman** [comment, marked as answer]
Hi there @uwufemboy ! 

That is an awesome suggestion and something we hope to look at so that it can help you with your daily work flow! 

I am going to create a tracking issue for this! Stay tuned to our future releases 🚀 Cheers

**Comment by eliperkins** [comment]
This has shipped on GitHub for iOS and is Coming Soon™ for GitHub for Android! 🎊 

Read more: https://github.blog/changelog/2022-11-15-github-for-ios-edit-files-from-browse-code/
